src/packaging/kpatch/commit_to_overlay.sh

```diff
@@ -417,14 +446,11 @@ for change in ${COMMIT_SHAS[@]}; do
 
   sed -r -i -e 's/^GITARGS=/GITARGS=\nGITAUTHOR ?= ""/g'        \
             -e 's/^GITARGS=/GITARGS=\nGITDATE ?= ""/g'          \
-            -e 's/commit -m/commit --file/g'                     \
-            -e 's/ commit / commit $(GITAUTHOR) $(GITDATE) /g'  \
-            -e 's/^(\s*&&).*format-patch.*/\1 echo committed/g' \
-            -e "s/ origin\/master / origin\/${KERNEL_BRANCH} /g" Makefile
+            -e 's/ commit / commit $(GITAUTHOR) $(GITDATE) /g' Makefile
```

Comment:
This will replace it globally, but you only need this in the new kernel-kpatch Makefile target. Could you make this more explicit to only catch the kernel-kpatch instance of 'commit' and not the existing kpatch one?

Reply:
Okay, looks good.